docs/common/craft-parts/reference/plugins/jlink_plugin.rst

```diff
@@ -0,0 +1,103 @@
+.. _craft_parts_jlink_plugin:
+
+JLink plugin
+=============
+
+The JLink plugin can be used for Java projects where you would want to
+deploy a Java runtime specific for your application or install a minimal
+Java runtime.
+
+
+Keywords
+--------
+
+This plugin uses the common :ref:`plugin <part-properties-plugin>` keywords as
+well as those for :ref:`sources <part-properties-sources>`.
+
+Additionally, this plugin provides the plugin-specific keywords defined in the
+following sections.
+
+jlink-jars
+~~~~~~~~~~~~~~~~~~
+**Type:** list of strings
+
+List of paths to your application's JAR files. If not specified, the plugin
+will find all JAR files in the staging area.
+
+Dependencies
+------------
+
+The plugin expects OpenJDK to be available on the system and to contain
+the ``jlink`` executable, unless a part named ``jlink-deps`` is defined.
+In that case, the plugin assumes that this part will stage the
```

Comment:
Not sure if my suggestion is correct, but "this part" is ambiguous.  It could refer to the part with the `jlink` plugin or the `jlink-deps` part.

```suggestion
In that case, the plugin assumes that the ``jlink-deps`` part will stage the
```